src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -178,12 +200,20 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
       : this.getUserEmail
   }
 
+  /** The Completing Party change message. */
+  private get changeCpMessage (): string {
+    const currentCpName = this.formatFullName(this.currentCompletingParty?.officer)
+    return `The Completing Party role was re-assigned to ${currentCpName}.\n` +
+      'This undo will restore the original Completing Party.'
+  }
+
   /**
    * Called when component is mounted.
    */
   private mounted (): void {
-    // initialize component 'changed' flag
-    this.setPeopleAndRolesChanged(false)
+    // initialize this component's 'valid' and 'changed' flags
+    this.setPeopleAndRolesValid(this.hasValidRoles && this.noMissingRoles)
+    this.setPeopleAndRolesChanged(this.hasChanges)
```

Comment:
Although this code does practically nothing in real life (ie, the watchers will update the flags), for testing we mount with props, and this code is used then.